keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/resource/TitusClusterHandlerTests.kt

```diff
@@ -141,6 +140,14 @@ class TitusClusterHandlerTests : JUnit5Minutests {
     spec = spec
   )
 
+  val exportable = Exportable(
+    account = spec.locations.account,
+    serviceAccount = "keel@spinnaker",
+    moniker = spec.moniker,
+    regions = spec.locations.regions.map { it.name }.toSet(),
```

Comment:
this is actually all regions in the account

Reply:
This is actually the same as       `regions = setOf(SimpleRegionSpec("us-east-1"), SimpleRegionSpec("us-west-2"))` , we are just refering to this map here.